flax/nnx/summary.py

```diff
@@ -151,14 +187,13 @@ def __repr__(self):
 
 
 def _to_dummy_array(x):
-  if isinstance(x,jax.ShapeDtypeStruct):
```

Comment:
Can you explain this change?

Reply:
Actually, that should be unnecessary in the current version of the PR. I'll remove it.